arangod/RestHandler/RestAdminLogHandler.cpp

```diff
@@ -113,13 +113,19 @@ RestStatus RestAdminLogHandler::execute() {
       return reportLogs(/*newFormat*/ true);
     } else if (suffixes.size() == 1 && suffixes[0] == "level") {
       handleLogLevel();
+    } else if (suffixes.size() == 1 && suffixes[0] == "structured") {
+      handleLogStructuredParams();
     } else {
       generateError(rest::ResponseCode::BAD,
                     TRI_ERROR_HTTP_SUPERFLUOUS_SUFFICES,
                     "superfluous suffix, expecting /_admin/log/entries");
     }
-  } else {
-    handleLogLevel();
+  } else if (type == rest::RequestType::PUT) {
+    if (suffixes.size() == 1 && suffixes[0] == "level") {
+      handleLogLevel();
+    } else {
```

Comment:
We should also check the number of suffixes here and the URL part "structured". Otherwise users could call PUT `/_admin/log/testi/1234` and it would work.

Reply:
Addressed in latest commit